surface/include/pcl/surface/mls.h

```diff
@@ -283,6 +306,18 @@ namespace pcl
       inline int
       getDilationIterations () { return dilation_iteration_num_; }
 
+      /** /brief Set wether the mls results should be stored for each point in the input cloud
+       *  /param[in] True if the mls results should be stored, otherwise false.
+       */
+      inline void
+      setCacheMLSResults(bool cache_mls_results) {cache_mls_results_ = cache_mls_results;}
```

Comment:
Missing space after function name and inside curly braces.
Also, I don't know how useful this will be, but in PCL we always have accessors for the algorithm configuration parameters. This would be `getCacheMLSResults()` in this case.

Edit: actually, this parameter can be modified during the algorithm execution, so I think this indeed will be a useful getter.